app/views/records/show_fields/_rights_note.html.erb

```diff
@@ -0,0 +1 @@
+<span itemprop="rights_note" class="detail-width-250"><%= record.solr_document.rights_note[0] %></span><br />
```

Comment:
@andjsmit The "[0]" goes away if this is a single-valued field, as per f642b02

Reply:
@andjsmit we probably also want to wrap this in a simple_format() or simple_format(auto_link()) call, is done for "description":
https://github.com/samvera/hyrax/blob/v2.9.6/app/views/hyrax/collections/_collection_description.erb#L2

Reply:
@aploshay Removed index. I did not ad simple_format as this messed display up by adding unwanted p and br tags and AFAIK this is not meant to be a link.

Reply:
@andjsmit are p, br tags unwanted if they are reflecting any newlines in the textbox value?  @nhomenda?

Reply:
@aploshay @nhomenda My concern was seeing an extra <p> and <br> tag being added to the content of the field. Turned out these extra tags came from pasting content into the field not as a result of wrapping the field. I've updated PR with simple_format wrap.